### PR TITLE
Add push for custom package definition

### PR DESCRIPTION
**Pull request by blaggacao**
Attempt to implement a push command for being able to push package definitions to the build root, in order to replicate custom packages. http://blog.docker.com/2013/06/create-light-weight-docker-containers-buildroot
See also: http://buildroot.uclibc.org/downloads/manual/manual.html#customize-packages

### CODE REVIEW

File: nano [hunk 1]
```diff
@@ -97,6 +97,29 @@ pull() {
...
+
+	case $1 in
+		package?(s))
+			local dir=packages; local files=packages/*//Config.in; 
+				local config=""; for f in $files; do; 
+				config="$config \n source $f"; done; 
+				[ -d $dir ] && cd $dir &&
+				docker cp . "$NANO_CONTAINER":/root/buildroot/package &&
+				echo Copied packages to $_ directory. &&
+				docker ssh "$NANO_CONTAINER echo "$config" >> /root/buildroot/Config.in &&
```
Comment (Bilge) [inline nano:114]: `docker ssh` does not appear to be a [documented command](https://docs.docker.com/engine/reference/commandline/cli/) even in the newest versions.

**Comment by Bilge** [comment]
I don't think defining specific targets on the local machine is particularly useful. A general purpose push command like `push <source> <destination>` might be useful, though.

File: nano (hunk 1 above)
Comment (blaggacao) [inline nano:114]: sorry, I mixed thinks up with docker-machine... My bad.

**Comment by blaggacao** [comment]
I went over the packages list and saw that postgres is meanwhile integrated into buildroot packages. So my usecase has actually gone.. We can close this, and I can reopen as I'm diving more into buildroot...

**Pull request closed without merging**